src/main/java/com/ivoyant/kafkaemailevent/service/EmailConsumer.java

```diff
@@ -28,6 +35,38 @@ public void consumeMessage(EmailDto emailDto) {
         sendEmail(emailDto);
     }
 
+    @KafkaListener(topics = "email-attach-event", groupId = "group2")
```

Comment:
Topic name and group ID should come from property file and use @Value.

Reply:
ok i will do changes thank you